aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -281,13 +283,54 @@ class CAFFE2_API OperatorHandle final {
 
 namespace detail {
 template<class... Args> inline void unused_arg_(const Args&...) {}
+
+template <typename T, std::enable_if_t<c10::impl::not_ok_to_box<T>::value>* = nullptr>
```

Comment:
oh so you're skipping arguments here? Can the observers handle that?

Reply:
for observers we don't need to convert all inputs, only the ones that can be converted (typically tensors, scalars)